unit/java_bytecode/java_utils_test.cpp

```diff
@@ -13,6 +13,7 @@
 
 #include <testing-utils/catch.hpp>
 
+#include <java_bytecode/java_types.cpp>
```

Comment:
including the `cpp` file here? is this the reason for the definitions in the `cpp` file instead of the header ?

Reply:
Vice-versa, including as declarations are not in the header, not in the header to avoid making the functions global. See earlier discussion and let me know if you're happy with my proposed solution (this will still include the cpp file)